app/assets/javascripts/dashboard.js

```diff
@@ -244,6 +244,13 @@ MinionPoller = {
     $("#download-kubeconfig").attr("disabled", !enabled);
   },
 
+  alertFailedBootstrap: function(message) {
+    if ($('.alert-message span').is(':empty'))
```

Comment:
Is missing braces the recommended style?

Reply:
Use curly braces always.

Reply:
fixed